app/controllers/api/members_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::MembersController < ApplicationController
+
+  def create
```

Comment:
Not sure a `rescue` block is necessary. It's typical to call `save` on the model which returns false if successful:

```ruby
member = Member.new(permitted_params) #use Rails' params whitelisting
respond_to do |format|
  if member.save
    member.send_to_ak
    format.json { render json: { member: member } }
  else
    format.json { render json: member.errors, status: :unprocessable_entity }
  end
end
```